components/sp-solutions/org.wso2.carbon.sp.solutions.apim.analytics/widgets/APIMApiCreatorAlertConfiguration/src/resources/widgetConf.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "APIM Api Creator Alert Configuration",
+    "id": "APIMApiCreatorAlertConfiguration",
+    "thumbnailURL": "",
+    "configs": {
+    	"pubsub": {
+      		"types": ["publisher"]
+    	},
+      
```

Comment:
remove extra spaces

Reply:
This was sample code added for revision and removed in 45ea197bf4dd881b1e9d79f5b3ba8d3cc12e05c9